packages/shared/src/types/abTests/banner.ts

```diff
@@ -86,3 +96,5 @@ export type BannerVariantFromTool = Omit<
 export type BannerTestFromTool = Omit<BannerTest, 'bannerChannel' | 'isHardcoded'> & {
     variants: BannerVariantFromTool[];
 };
+
+export type BannerDesignFromTool = { name: string } & ConfigurableDesign;
```

Comment:
what's the reasoning behind this separate `BannerDesignFromTool` type?

Reply:
My thinking was that there'd be data in the Dynamo table that was no longer interesting once the design props have been expanded out and added to the variant (things like the name, and possibly other metadata in future). Maybe that's not a useful distinction though and there'd be no harm in keeping those props on the variant design. What do you think? Shall I reduce this down to a single type?